nnpdf_data/nnpdf_data/commondata/ATLAS_WPWM_7TEV_46FB/filter.py

```diff
@@ -6,6 +6,11 @@
 import yaml
 from filter_utils import get_data_values, get_kinematics, get_systematics
 
+def prettify_float(representer, value):
+    return representer.represent_scalar('tag:yaml.org,2002:float', '{:.4f}'.format(value))
+
```

Comment:
Hi @ecole41 you can import this from nnpdf_data filter utils


```
from nnpdf_data.filter_utils.utils import prettify_float
```

Reply:
Thank you, I'll change that now